hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/EntityIdUtils.java

```diff
@@ -351,4 +351,12 @@ public static EntityNum unaliased(
             return EntityNum.fromAccountId(idOrAlias);
         }
     }
+
+    public static boolean isValidSizeEvmAddress(final ByteString key) {
+        return key.size() == EVM_ADDRESS_SIZE;
+    }
+
+    public static boolean isValidSizeECDSAAddress(final ByteString key) {
```

Comment:
Would naming the methods like `hasEvmAddressSize(alias)` / `isOfEvmAddressSize(alias)` be clearer?

Reply:
second suggestions sounds clearer, thank you !